python/fledge/services/core/api/auth.py

```diff
@@ -872,6 +875,51 @@ async def delete_user(request):
     return web.json_response({'message': "User has been deleted successfully."})
 
 
+@has_permission("admin")
```

Comment:
Maybe this should return the certificate (cert file) such that the user can save it rather than having a separate download step.

Reply:
API has now been updated to retrieve the file content.